src/Pages/SJSUDiscordBot/GoogleLogin.js

```diff
@@ -0,0 +1,128 @@
+import React, {useState} from 'react';
+import {GoogleLogin,
+  GoogleLogout} from 'react-google-login';
+import { GOOGLE_API_CLIENT_ID } from '../../config/config.json';
+import { getTempUser, addUser } from '../../APIFunctions/GoogleLoginPage.js';
+
+function Verify(){
+  const [isLogined, setLoginState] = useState(false);
+  const [accessToken, setAccessToken] = useState('');
+  const [username, setUsername] = useState('');
+  const [discordID, setDiscordID] = useState(0);
+  const [id, setID] = useState(window.location.href.split('/')[5]);
+  const [validID, setValidID] = useState(false);
+  const [loginFailed, setLoginFailed] = useState(false);
+  const [logoutFailed, setLogoutFailed] = useState(false);
+  const [SJSUEmail, setSJSUEmail] = useState(false);
+
+  const getUser = async () => {
+    await getTempUser(id).then(res => {
+      if(!res.error){
+        setUsername(res.responseData.username);
+        setDiscordID(res.responseData.id);
+        setValidID(true);
+      }
+    });
+  };
+
+  React.useEffect(() => {
+    getUser();
+  }, []);
+
+  const onSuccess = async (res) => {
+    if(res.accessToken){
+      setLoginState(true);
+      setAccessToken(res.accessToken);
+      if(res.profileObj.email.includes('sjsu.edu')){
+        setSJSUEmail(true);
+        const body = {
+          discordID: discordID,
+          googleTokenId: res.tokenId
+        };
+        await addUser(body);
+      }
+    }
+  };
+
+  const onLogout = (res) => {
+    setLoginState(false);
+    setAccessToken('');
+  };
+
+  const onLoginFailure = (res) => {
+    setLoginFailed(true);
+  };
+
+  const onLogoutFailure = (res) => {
+    setLogoutFailed(true);
+  };
+
+  return(
+    <div>
+      <section style={{margin: '100px'}}>
+        <h1>SJSU SCE Discord Verify</h1>
+        { !validID?
+          <h5>Invalid ID: please type verify in the discord chat again</h5>
+          : isLogined ?
+            logoutFailed ?
+              <>
+                <h5>Logout failed. Please try again or contact us. </h5>
```

Comment:
```suggestion
                <h6>Logout failed. Please try again or contact us. </h6>
```
Nested headings should be a higher level.